local-cli/runAndroid/runAndroid.js

```diff
@@ -263,17 +263,18 @@ function runOnAllDevices(args, cmd, packageNameWithSuffix, packageName, adbPath)
 }
 
 function startServerInNewWindow(port) {
-  const scriptFile = /^win/.test(process.platform) ?
+  const isWin = /^win/.test(process.platform);
```

Comment:
Please, don't use abbreviations.

Reply:
Renamed `isWin` to `isWindows` everywhere where it's necessary.